src/strings-in-block/class-html.php

```diff
@@ -174,4 +174,38 @@ private function get_dom_handler( \WP_Block_Parser_Block $block ) {
 
 		return new StandardBlock();
 	}
+
+	/**
+	 * @param                        $text
+	 * @param                        $lang
+	 * @param \WP_Block_Parser_Block $block
+	 * @param array                  $string_translations
+	 * @param                        $element
+	 * @param                        $dom_handle
+	 *
+	 * @return \WP_Block_Parser_Block
+	 */
+	private function updateTranslationInBlock( $text, $lang, \WP_Block_Parser_Block $block, array $string_translations, $element, $dom_handle ) {
+		$translation = $this->getTranslation( $text, $lang, $block, $string_translations );
+		if ( $translation ) {
+			$block = self::update_string_in_innerContent( $block, $element, $translation );
+			$dom_handle->setElementValue( $element, $translation );
+		}
+
+		return $block;
+	}
+
+	private function getTranslation( $text, $lang, \WP_Block_Parser_Block $block, array $string_translations ) {
```

Comment:
@brucepearson, don't you think we could also use this method for block attributes?

https://github.com/OnTheGoSystems/wpml-page-builders-gutenberg/blob/7d1dc300f95fdacefca3534c47093abf2662e527/src/strings-in-block/class-attributes.php#L161

I don't know if it's yet possible to have a shortcode inside a block attribute but I guess it will come soon :)

This `getTranslation` method could go in the parent class.

Reply:
@strategio I still have to go through all Toolset blocks. I'll add when/if it's needed.